docs/operations/operations-import-ALZAccount.md

```diff
@@ -0,0 +1,206 @@
+# Operations - How to migrate an Amazon Landing Zone (ALZ) account "as is" into an Amazon Secure Environment Accelerator.
+
+<!-- TOC depthFrom:1 depthTo:4 -->
+
+- [1. Prerequisites (Setup)](#1-prerequisites-setup)
+- [2. Landing Zone - Disassociate the account from the Landing Zone](#2-landing-zone---disassociate-the-account-from-the-landing-zone)
+- [3. Landing Zone - Remove the account from the landing Zone (ALZ) organizations and make standalone](#3-landing-zone---remove-the-account-from-the-landing-zone-alz-organizations-and-make-standalone)
+- [4. Accelerator (ASEA) - Invite the account into it's organization](#4-acceleartor-asea---invite-the-account-into-its-organization)
+- [5. Accelerator (ASEA) - Move the linked account from the top level root OU into the appropriate OU managed by the ASEA](#5-acceleartor-asea---move-the-linked-account-from-the-top-level-root-ou-into-the-appropriate-ou-managed-by-the-asea)
+- [6. Accelerator (ASEA) - Verify access control with roles, SSO, etc](#6-acceleartor-asea---verify-access-control-via-roles-sso-etc)
+- [7. Landing Zone - Close down the remainder of the Landing zone core accounts and then the mgmt. account](#7-landing-zone---close-down-the-remainder-of-the-landing-zone-core-accounts-and-then-the-mgmt-account)
+
+# Purpose
+
+This document describes the steps to migrate an existing Linked account from an Amazon Landing Zone (ALZ) to an Amazon Secure Environment Accelerator (ASEA).
+
+## 1. Prerequisites (Setup)
+
+### 1.0.
+
+Setup and run initial tests on the ASEA with SSO and permission sets with an account under the appropriate OU (where the linked account will be migrated to).  Ensure that AWS SSO is properly configured to have the permission sets and correct permissions for the team whose account is being migrated over. This would include configuration of the ASEA’s AWS Managed Active Directory (MAD) which should align with how the team migrating their account has their AWS SSO and MAD configured today.
+
+### 1.1.
+
+If working with your AWS account team (TAM/SA), they will reach out to an internal team to request the linked account be switched to invoicing.  This way the customer doesn’t have to enter a credit card when making the account standalone in the upcoming steps.
+
+### 1.2.
+
+Confirm access to the email account associated to the ALZ linked account being migrated to the ASEA, as well as access to the AWS linked account itself.  This runbook will first make the account standalone (remove from ALZ organizations) so one needs to make sure the customer has root access to the account and that the email address attached to the account is available before running this procedure.  If needed, you can reset the linked account root password by following: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_passwords_change-root.html
```

Comment:
", as well as access to the AWS linked account itself" - what does this mean.  also grammar/the way tacked onto sentence is awkward.

Reply:
Done - made the changes  and had one commit with spelling, grammar, added some more subtitles and tried to simplify where possible.   Please feel free to edit changes directly.